index.bs

```diff
@@ -4091,6 +4104,138 @@ The following extended attributes are applicable to [=iterable declarations=]:
 </pre>
 
 
+<h4 id="idl-async-iterable">Asynchronously iterable declarations</h4>
```

Comment:
Nit: I've never seen it spelled out as "asynchronously iterable". Maybe an "asynchronous iterable", as in the title of https://github.com/tc39/proposal-async-iteration. But "async iterable" is much more common.